tests/ctst/steps/pra.ts

```diff
@@ -71,6 +72,7 @@ async function installPRA(world: Zenko, sinkS3Endpoint = 'http://s3.zenko.local'
         // prometheusHostname: 'prom.dr.zenko.local', // could be any name, cert will be auto-generated
         prometheusExternalIpsDiscovery: true,
         prometheusDisableTls: true,
+        forceRotateServiceCredentials: true,
```

Comment:
should not be set on "initial" install : only use on reinstall

Reply:
btw should just be removed, this is unrelated to this PR and will conflict with one about `uninstall`